zcash_primitives/src/transaction/builder.rs

```diff
@@ -398,9 +405,10 @@ impl<'a, P: consensus::Parameters> Builder<'a, P, ()> {
     /// sent represents the total steps completed so far. It will eventually send number
     /// of spends + outputs. If there's an error building the transaction, the channel is
     /// closed.
+    #[cfg(feature = "std")]
     pub fn with_progress_notifier(
         self,
-        progress_notifier: Sender<Progress>,
+        _progress_notifier: Sender<Progress>,
```

Comment:
This is never unused.
```suggestion
        progress_notifier: Sender<Progress>,
```

Reply:
This causes unused variable warnings, because `progress_notifier` is only used when the `std` feature is enabled.